src/index.ts

```diff
@@ -1,19 +1,70 @@
 import { Probot } from "probot";
 import fs from 'fs';
-const k8s = require('@kubernetes/client-node');
+import k8s, { V1Secret } from '@kubernetes/client-node';
+const { createAppAuth } = require("@octokit/auth-app");
+require('dotenv').config();
 
+
+const EXPIRATION_THRESHOLD = 5 * 60000;
+
+// K8s client for using k8s apis.
 const kc = new k8s.KubeConfig();
 kc.loadFromDefault();
 const k8sApi = kc.makeApiClient(k8s.CoreV1Api);
 const k8sContext = kc.getCurrentContext();
-const namespace = (() => {
+const k8snamespace = (() => {
   if (k8sContext === 'inClusterContext') {
-    return fs.readFileSync('/var/run/secrets/kubernetes.io/serviceaccount/namespace', 'utf8');
+    return fs.readFileSync('/const/run/secrets/kubernetes.io/serviceaccount/namespace', 'utf8');
```

Comment:
This path is still `/var` not `/const` :smile:

Reply:
changed :+1: